aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -2175,6 +2187,56 @@ std::tuple<Tensor, Tensor> _symeig_helper_cuda(const Tensor& self, bool eigenvec
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ linalg_eigh ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+// This is a type dispatch function for 'apply_magma_eigh'
+void linalg_eigh_magma(Tensor& eigenvalues, Tensor& eigenvectors, Tensor& infos, bool upper, bool compute_eigenvectors) {
+  // MAGMA just calls LAPACK for n <= 128
+  // See https://bitbucket.org/icl/magma/src/e6fdca447bd402693e8b0b950a898b6879bbcc41/src/zheevd_gpu.cpp?at=master#lines-258
+  // in addition it does bad stuff ignoring lda breaking 0x0 inputs
+  bool use_magma = true;
+  if (eigenvectors.size(-1) <= 128) {
+    use_magma = false;
+  }
+
+  if (use_magma) {
```

Comment:
Write the condition directly here so it's more obvious `use_magma` is only needed here.